sdks/python/apache_beam/utils/options.py

```diff
@@ -235,7 +235,8 @@ def _add_argparse_args(cls, parser):
                         default=None,
                         help='GCS path for staging code packages needed by '
                         'workers.')
-    # Remote execution must check that this option is not None.
```

Comment:
Remote execution comment can stay. Since temp_location may still be None if staging_location is None.

And the new comment could be written without referencing Dataflow. (e.g. "If temp_location is not set, it defaults to staging_location.")